Source/ASCollectionNode.h

```diff
@@ -291,6 +315,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)waitUntilAllUpdatesAreProcessed;
 
+/**
+ *  Returns YES if the ASCollectionNode contents are completely synchronized with the underlying collection-view layout.
+ */
+@property (nonatomic, readonly) BOOL isSynchronized;
+
+/**
+ *  Schedules a block to be performed (on the main thread) as soon as the completion block is called
+ *  on performBatchUpdates:.
+ *
+ *  When isSynchronized == YES, the block is run block immediately (before the method returns).
+ */
+- (void)onDidFinishSynchronizing:(nullable void (^)(void))didFinishSynchronizing;
```

Comment:
Nit: This decl says `nullable` but the impl will crash with a nil block.

Probably best to let this argument default to `nonnull` since I can't think of a use case for knowingly passing nil in here and _also_ add a nil check in the implementation.

Reply:
@Adlai-Holler cool, fixed all definitions (and also made the change for the existing onDidFinishProcessingUpdates:).

I moved Synchronizing to +Beta for now. A better name might be onDidFinishSynchronizingLayout, or ...LayoutSynchronization. Or even merging with onDidFinishProcessingUpdates with the option to specify a type of "finished", with synchronized including the end of animation. It's all a bit subtle and confusing, but having this call point available has been very useful even as the name remains a bit unclear.